discourse_migration/userguide.md

```diff
@@ -0,0 +1,155 @@
+# User guide for the migration of the mailing lists
+
+This document is intended to help LLVM users to migrate from the mailing lists to
+Discourse. Discourse two basic ways for interaction: Via the [web
+UI](https://llvm.discourse.group/) and via emails.
+
+## Setting up your account
+
+The easiest way is to create an account using your GitHub account:
+
+1. Navigate to https://llvm.discourse.group/
+1. Click on "Sign Up" in the top right corner.
+1. Choose "With GitHub" on the right side and log in with your GitHub account.
+
+## Structure of Discourse
+
+Discourse's structure is similar to a set of mailing lists, however different
+terms are used there. To help with the transition, here's a translation table
+for the terms:
+
+| Mailing list | Discourse |
+|--------------|-----------|
+| *Mailing list*, consists of threads | *category*, consists of topics |
+| *thread*, consists of emails | *topic*, consists of posts |
+| *email* | *post* |
+
+## Setting up email interactions
+
+Some folks want to interact with Discourse purely via their email program. Here
+are the typical use cases:
+
+* You can [subscribe to a category or topic](https://discourse.mozilla.org/t/how-do-i-subscribe-to-categories-and-topics/16024)
+* You can reply to a post, including quoting other peoples texts
+  ([tested](https://llvm.discourse.group/t/email-interaction-with-discourse/3306/4) on GMail).
+* [Quoting previous topics in an reply](https://meta.discourse.org/t/single-quote-block-dropped-in-email-reply/144802)
+* **TODO:** Creating new topics via email is
+  [supported](https://meta.discourse.org/t/start-a-new-topic-via-email/62977)
+  but not configured at the moment. We would need to set up an email address
+  per category and give Discourse POP3 access to that email account. This sounds
+  like a solvable issue.
+* You can filter incoming emails in your email client by category using the
+  `List-ID` email header field.
+
+## Mapping of mailing lists to categories
```

Comment:
It might be worth noting here that we have a lot more specific categories on Discourse than we previously had mailing lists for. Does it make sense to mention all the "new" categories that were previously discussed on the same mailing list? Or at least mention it in writing. In my opinion this is an organizational benefit of Discourse.

Reply:
Good point, can you propose something? I have to admit, I don't really have a good overview of the mapping.

I merged the file to the main branch, feel free to create a PR for that.